gen-protobuf.sh

```diff
@@ -1,22 +1,33 @@
 #!/usr/bin/env bash
 
+set -e
+
+rm -rf message avatica-tmp
+
 export AVATICA_VER="rel/avatica-1.10.0"
 
-rm -rf message
-rm -rf avatica-tmp
+mkdir -p avatica-tmp
+pushd avatica-tmp &> /dev/null
 
-git init avatica-tmp
-cd avatica-tmp
-git remote add origin https://github.com/apache/calcite-avatica/
+git init
+git remote add origin https://github.com/apache/calcite-avatica.git
 git config core.sparsecheckout true
 echo "core/src/main/protobuf/*" >> .git/info/sparse-checkout
+
 git fetch --depth=1 origin $AVATICA_VER
 git checkout FETCH_HEAD
 
-cd ..
-mkdir message
-protoc --proto_path=avatica-tmp/core/src/main/protobuf/ --go_out=import_path=message/message avatica-tmp/core/src/main/protobuf/*.proto
+popd &> /dev/null
+
+mkdir -p message
+
+perl -pi -e 's/^(option java_package.*)/$1\noption go_package = "message";/' avatica-tmp/core/src/main/protobuf/*.proto
```

Comment:
Also, another question, why do we need to do this here?

Reply:
I'm not sure what version of `protoc` that was in use previously, but without specifying the `go_package` option, you get an error like this:

```
2017/10/16 13:12:23 protoc-gen-go: error:inconsistent package names: requests common
```

There might be a better/easier way to fix this via a passed command line option, but the most expedient/easiest thing I found to do was to just manually set the `go_package` option in the protobuf files.

Reply:
hey @kenshaw,

Thanks for investigate. I did some testing and the following works:

```
protoc --proto_path=avatica-tmp/core/src/main/protobuf --go_out=import_path=message:message avatica-tmp/core/src/main/protobuf/*.proto
```
Can you update the PR to use this? We should be good to merge after this change 👍

Reply:
Done.